addon/components/select-2.js

```diff
@@ -411,6 +430,21 @@ var Select2Component = Ember.Component.extend({
     this.watchDisabled();
   },
 
+  createRecordFromTag: function(tag) {
+    var type;
+    if (this.get('value') instanceof Ember.ArrayProxy) {
+      type = this.get('value.content.type');
+    } else {
+      type = this.get('value.type');
+    }
+
+    var record = this.get('container').lookup('store:main').createRecord(type.typeKey);
+
+    record.set(this.get('optionLabelPath'), tag[this.get('optionLabelPath')]);
+
+    return record;
+  },
+
```

Comment:
Yeah, I can add `create-tag` and `remove-item` actions to this component? If you implement them, it will skip the default behaviour. Something like that?